src/currency/formatter/fn.js

```diff
@@ -0,0 +1,32 @@
+define([
+	"../../common/validate/parameter-presence",
+	"../../common/validate/parameter-type/number",
+	"../name-format"
+], function( validateParameterPresence, validateParameterTypeNumber, currencyNameFormat ) {
+
+return function( numberFormatter, pluralGenerator, properties ) {
+	var fn;
+
+	// Return formatter when style is "code" or "name".
+	if ( pluralGenerator && properties ) {
```

Comment:
Yes, it would. Note this function is returned by the formatter generator. So, doing it properly takes a little more than just splitting this into two files. Or you'd end up still having the `if` condition in the parent function and having actually no much gain.